vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -75,7 +75,11 @@ static const std::map<std::string, bool> kSupportedRegions = {
   { "ZA", false }   // South Africa
 };
 
-static char kUntargetedPageClassification[] = "untargeted";
+#if defined(OS_ANDROID)
+const int kMaximumAdNotifications = 3;
```

Comment:
Should be platform agnostic and returned from platform specific classes, please see previous comments

Reply:
Issue created for that https://github.com/brave/brave-browser/issues/6200